python_modules/libraries/dagster-dbt/dagster_dbt/dagster_dbt_translator.py

```diff
@@ -11,17 +11,17 @@
 
 class DagsterDbtTranslator:
     @classmethod
-    def node_info_to_asset_key(cls, node_info: Mapping[str, Any]) -> AssetKey:
-        """A function that takes a dictionary representing the dbt node and returns the
-        Dagster asset key the represents the dbt node.
+    def get_asset_key(cls, dbt_resource_info: Mapping[str, Any]) -> AssetKey:
```

Comment:
I think it's worth adding a sentence that explicitly calls out that a dbt resource is different than a Dagster resource.

Reply:
yeah, we should update the API docstrings here.

also maybe consider dbt_resource_props, since dbt resource info in google yields therapy as the first search result, and i don't want to compete against that when we have documentation on these concepts soon